src/components/Cards/CardEvent/CardEvent.scss

```diff
@@ -15,6 +16,7 @@
     justify-content: center;
     width: 4.5rem;
     height: 4.5rem;
+    border-radius: 0px 0px 4px 0px;
```

Comment:
this seems weird why one corner?

Reply:
![image](https://user-images.githubusercontent.com/29209916/108554005-56b97900-72a8-11eb-89b9-9ffee0d90d17.png)
all 4 corners

Reply:
![image](https://user-images.githubusercontent.com/29209916/108554071-6cc73980-72a8-11eb-894c-e485e0645c57.png)